packages/server/app/Controllers/Http/UsersController.ts

```diff
@@ -0,0 +1,98 @@
+import { HttpContextContract } from '@ioc:Adonis/Core/HttpContext';
```

Comment:
In the last [last merged PR](https://github.com/AdonisEnProvence/MusicRoom/commit/9b8075fcd126181f126cf397e4fc5770f5b58193), we've been creating new controller especially the `UserProfileController` I don't know if we wanna merge both in one ?

Reply:
We can, indeed!
I created this controller before we merged your PR.

Reply:
Great !